### ISSUE TITLE
Wrong directory in /usr/lib/vdr/bin/shutdown.sh

### ISSUE THREAD

**Issue by soundscrazy** [VDR4Arch/vdr4arch#191]
There is a typo in /usr/lib/vdr/bin/shutdown.sh

for file in /usr/lib/vdr/shutdown-hooks/*; do 
.. should be (if the typo is not in the directory name hopefully ;-)):
for file in /usr/lib/vdr/shutdown_hooks/*; do
(see underline)

Thx anyway

**Comment by soundscrazy** [comment]
.. the star has been lost in the for loop when pasting, so: 
hooks/
should be:
hooks/*
in every line where this phrase occurs

**Comment by utiltiy** [comment]
The path is here on my arch system:

> /usr/share/vdr/shutdown-hooks/

**Comment by soundscrazy** [comment]
definition in shutdown-wrapper.c:
#define REAL_PATH "/usr/lib/vdr/bin/shutdown.sh"

lines of /usr/lib/vdr/bin/shutdown.sh:
#Run through shutdown-hooks
for file in /usr/lib/vdr/shutdown-hooks/*; do

in my arch installation the folder name there is:
/usr/lib/vdr/shutdown_hooks

and yes, this folder also exists - but is also not used by shutdown.sh:
/usr/share/vdr/shutdown-hooks

**Comment by utiltiy** [comment]
I don't understand your problem. My system have no folder in /usr/lib/vdr/bin, only:
- shutdown-wrapper
- shutdown.sh
It seems you have another problem, mixed packages or anything else....

**Comment by M-Reimer** [comment]
Exactly. Something is wrong on your system. The only package, we actually support to use "shutdown-hooks" is:
https://github.com/VDR4Arch/vdr4arch/blob/vdr-stable/vdr-addon-lifeguard/PKGBUILD
And this package places the "hook" to
/usr/lib/vdr/shutdown-hooks/
So: For "vdr4arch" *currently* the correct path for "shutdown hooks" *is* /usr/lib/vdr/shutdown-hooks/
I don't know where you got the other directory from.

**Comment by soundscrazy** [comment]
Hi, sorry for this.
I did a clean install with just a few plugins needed for a server installation (no lifeguard) and thought it's a problem with one package - actually I don't know where this folder comes from (I didn't create it myself). I just stumbled on it after editing my own shutdown script in this folder an it didn't get called.
So I checked the script an found the different destination.
I will change the path and assume there  is something wrong with my system.
Thx for looking at it.
